modules/contracts/apps/subscription/src/queries.rs

```diff
@@ -0,0 +1,5 @@
+// pub fn handle_value_query(deps: Deps, query: ValueQueryMsg) -> StdResult<Binary> {
+//     query_assets_from_mem(deps, ans_host_addr, '')
+//     if query.asset_info.equal(asset)
+
+// }
```

Comment:
Can we remove this file?

Reply:
I need to check my vision, haven't even seen this file